var/spack/repos/builtin/packages/freesasa/package.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Freesasa(AutotoolsPackage):
+    """C-library for calculating Solvent Accessible Surface Areas"""
+
+    homepage = "http://freesasa.github.io"
+    url = "https://github.com/mittinatten/freesasa/archive/refs/tags/2.1.2.tar.gz"
```

Comment:
Can we use [this artifact](https://github.com/mittinatten/freesasa/releases/download/2.1.2/freesasa-2.1.2.zip) to have a stable shasum to check?

Reply:
Is there an advantage in using the artefact instead of the `Source code (tar.gz)`? I've found instances (in other projects) where such artifacts were missing, while `Source code (tar.gz)` is always there by default.